homework.js

```diff
@@ -0,0 +1,212 @@
+//Q1
+
+onst  summation = function (n) {
+  let output = 0;
+  for( i = 1;  i <= n; i++) {
+    output = output  + i;
+  }
+  return output ;
+};
+
+let n = 5 ;
+console.log('Sum of numebers from 1 to '+ n +' is '+ summation(n));
+
+
+//Q2
+
+const summationEven = function (n){
+  let sum = 0 ;
+ 
+ for ( i=0; i <=n; i=i+2) {
+   sum +=1
+   }
+ return sum ;
+ 
+ };
+  let n = 6
+ console.log(summationEven(n));
```

Comment:
this function counts the even values from 0 to the input value, it doesn't sum them.